README.md

```diff
@@ -1 +1,61 @@
-# terraform-aws-named-subnets
\ No newline at end of file
+# terraform-aws-named-subnets
+
+Terraform module for named `subnets` provisioning.
```

Comment:
Perhaps we should link to this for an explanation of the math: https://github.com/cloudposse/terraform-aws-dynamic-subnets#tldr